tracer/build/_build/Build.Steps.Debugger.cs

```diff
@@ -27,6 +28,8 @@ partial class Build
 
     Project DebuggerSamples => Solution.GetProject(Projects.DebuggerSamples);
 
+    Project ExceptionReplaySamples => Solution.GetProject(Projects.ExceptionReplaySamples);
```

Comment:
Shouldn't the name start with `Debugger` for consistency?

Reply:
I don't think so. We use the name `Debugger` interchangeably with `Dynamic Instrumentation`, thus the `Debugger` is prefixed for the other projects. Exception Replay is a _separate_ product (though most of the functionality relies on the instrumentation of Dynamic Instrumentation). I prefer to keep Exception Replay separate from Debugger/Dynamic Instrumentation to avoid confusions. They don't rely on one another (Exception Replay can be enabled with/without DI).
Does that make sense?

Reply:
Not enough meat to go that far 🤓

Reply:
I'm a big fun of consistency. IMO, If you treat `ExceptionReplay` as separate product, it should have all the CI tasks of itself. If you treat it as part of `Dynamic Instrumentation`, because of "not enough meat", then you probably should have the same naming convention.

I'm fine with any of it.